graphql_compiler/tests/schema_transformation_tests/test_rename_schema.py

```diff
@@ -786,6 +768,14 @@ def test_field_in_list_still_depends_on_suppressed_type(self):
     def test_rename_using_dict_like_prefixer_class(self):
         class PrefixNewDict(object):
             def get(self, key, default=None):
+                if key == "Date" or key in specified_scalar_types:  # pylint: disable=E1135
+                    # pylint produces a false positive when checking for membership for
+                    # specified_scalar_types-- see issue here:
+                    # https://github.com/PyCQA/pylint/issues/3743
+                    # specified_scalar_types is a FrozenDict whose keys are built-in scalar type
+                    # names (e.g. "String")
+                    return key  # Making an exception for Date and all built-in scalar types because
+                    # they are scalars and renaming and suppressing them hasn't been implemented yet
```

Comment:
Don't we have other custom scalars in addition to `Date`? I'm not sure why `Date` in particular is being special-cased, it would be good to explicitly mention that.

Also, I find this comment hard to read. How about this formatting instead:
```suggestion
                    # names (e.g. "String")
                    #
                    # Making an exception for Date and all built-in scalar types because
                    # they are scalars and renaming and suppressing them hasn't been implemented yet
                    return key  
```
or perhaps even better, code explanation first, pylint suppression comment and description after?

Reply:
`Date` is the only scalar used in the schema-- however, after I wrote this code, I found a better way to do this that doesn't require any pylint suppressions or hard-coded values.